app/models/user.py

```diff
@@ -0,0 +1,43 @@
+from app.controllers import user
+
+class User:
+    """
+    Manages a user (given its id) and will obtain data from database as
+    appropriate.
+    
+    Note:
+        No checks are done to check if `id` actually exists.
+        You can also use this to create a user, if this is the case, don't call
+        on any method that would otherwise result in an error.
+    """
+    
+    def __init__(self, id, name=None):
+        """
+        If you don't know some of the data, don't provide it and it will be
+        fetched from the database.
+        
+        Args:
+            id (int): User id of the user
+            name (:obj:`str`, optional): Name of the user if known
+        """
+        
+        self.id = id
+        self.__name = name
+    
+    def get name(self):
```

Comment:
Function names can't have spaces. I'm guessing this is supposed to be `get_name`.